arbeitszeit_web/api/authentication.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+
+from arbeitszeit_web.api.response_errors import Unauthorized
+from arbeitszeit_web.session import Session
+
+
+@dataclass
+class Authenticator:
+    session: Session
+
+    @property
+    def authenticate_request(self) -> None:
```

Comment:
This is not a property but a method. I think that the `@property` decorator obfuscates the fact that `authenticate_request` has a side effect.